go/vt/schemadiff/column.go

```diff
@@ -137,6 +147,17 @@ func (c *ColumnDefinitionEntity) ColumnDiff(
 				c.columnDefinition.Type.Options.Collate = env.CollationEnv().LookupName(collation)
 			}
 		}
+		// other column definition
```

Comment:
Ah, I see. So this and other are the two things being diffed. The other related comments are still worthwhile though IMO.

Reply:
I've updated this and other comments.